inscriptions/ISic001380.xml

```diff
@@ -192,33 +197,22 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140886">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#TNS">
                 <ab>
-                    <lb n="1"/>
-                    <supplied reason="lost">ἡ φιλότ</supplied>εκνος σοφὴ
-                    <supplied reason="lost">— —</supplied>
-                    <lb n="2"/>
-                    <supplied reason="lost">— — — —</supplied>ς μακαριοισ
-                    <supplied reason="lost">— —</supplied>
-                    <lb n="3"/>
-                    <supplied reason="lost">— — — —</supplied>
-                    κρίσιν ΚΑ
-                    <unclear>Ι</unclear>
-                    <supplied reason="lost">— —</supplied>
-                    <lb n="4"/>
-                    <supplied reason="lost">— — — — —</supplied>ΗΝ <surplus>²piscis</surplus>²
-                    <lb n="5"/>
+                    <lb n="1"/><supplied reason="lost">ἡ φιλότ</supplied>εκνος σοφὴ<gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="2"/><gap reason="lost" extent="unknown" unit="character"/><orig>Σ</orig> μακαριοισ<gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="3"/><gap reason="lost" extent="unknown" unit="character"/> κρίσιν <orig>ΚΑ<unclear>Ι</unclear></orig><gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="4"/><gap reason="lost" extent="unknown" unit="character"/><orig>ΗΝ</orig> <g ref="#fish"/>
```

Comment:
add earlier in the file, after 'handDesc', but before end of 'physDesc':
`<decoDesc><decoNote ana="#decoration.figure.animal">image of a fish appears lower right at end of text</decoNote></decoDesc>`